src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -623,15 +656,18 @@ void testSelectStarConvertedToColumnsListJoinReversed() {
         tableRight.insert("ON", "R2_1", "R3_1");
```

Comment:
I don't get what the test with a constant JOIN criteria is supposed to prove.

Reply:
This is a test for a correctly formed list of columns when we use SELECT * (We're re-writing * to an explicit list). So we test that the columns are in the right order and the values are correct